File: C#/printCODE39.cs
using System;
using System.Collections.Generic;
using System.IO.Ports;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GertecEasyPrint
{
    internal class printCODE39 {
        public static String PrintCODE39(String codeText) {
            return "\x1d\x6b\x04\x43\x4f\x44\x45\x20\x33\x39\x00" + codeText;
        }
    }
}

File: CPP/printCODE39.cpp
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

string printCODE39(string texto) {

    string codebar_code39 = {0x1D,0x6B,0x04,0x45,0x00};
    
    return codebar_code39 + texto;
}